kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTaskTest.java

```diff
@@ -297,6 +297,36 @@ public void testPutWhenPartitioningOnMessageTimeWhenNoTimestampType() {
         TimestampType.NO_TIMESTAMP_TYPE, null)));
   }
 
+  @Test(expected = BigQueryConnectException.class)
+  public void testSimplePutException() throws InterruptedException {
+    final String topic = "test-topic";
+
+    Map<String, String> properties = propertiesFactory.getProperties();
+    properties.put(BigQuerySinkConfig.TOPICS_CONFIG, topic);
+    properties.put(BigQuerySinkConfig.DATASETS_CONFIG, ".*=scratch");
+
+    BigQuery bigQuery = mock(BigQuery.class);
+    Storage storage = mock(Storage.class);
+
+    SinkTaskContext sinkTaskContext = mock(SinkTaskContext.class);
+    InsertAllResponse insertAllResponse = mock(InsertAllResponse.class);
+    when(bigQuery.insertAll(any())).thenReturn(insertAllResponse);
+    when(insertAllResponse.hasErrors()).thenReturn(true);
+    when(insertAllResponse.getInsertErrors()).thenReturn(Collections.singletonMap(
+       0L, Collections.singletonList(new BigQueryError("no such field", "us-central1", ""))));
+
+    SchemaRetriever schemaRetriever = mock(SchemaRetriever.class);
+    SchemaManager schemaManager = mock(SchemaManager.class);
+
+    BigQuerySinkTask testTask = new BigQuerySinkTask(bigQuery, schemaRetriever, storage, schemaManager);
+    testTask.initialize(sinkTaskContext);
+    testTask.start(properties);
+
+    testTask.put(Collections.singletonList(spoofSinkRecord(topic)));
+    Thread.sleep(100);
+    testTask.put(Collections.singletonList(spoofSinkRecord(topic)));
```

Comment:
It shouldn't be necessary to give it another record, right? Think we can just use:
```suggestion
    testTask.put(Collections.emptyList());
```